project/AkkaBuild.scala

```diff
@@ -42,40 +42,80 @@ object AkkaBuild extends Build {
-      Dist.distExclude := Seq(actorTests.id, docs.id, samples.id, osgi.id),
+      Dist.distExclude := Seq(
+        actorTests.id,
+        benchJmh.id,
+        docs.id,
+        httpTests.id,
+        persistenceShared.id,
+        remoteTests.id,
+        samples.id,
+        streamTests.id,
+        streamTestsTck.id,
+        osgi.id
+      ),
       S3.host in S3.upload := "downloads.typesafe.com.s3.amazonaws.com",
       S3.progress in S3.upload := true,
       mappings in S3.upload <<= (Release.releaseDirectory, version) map { (d, v) =>
         val downloads = d / "downloads"
         val archivesPathFinder = downloads * s"*$v.zip"
-        archivesPathFinder.get.map(file => (file -> ("akka/" + file.getName)))
+        archivesPathFinder.get.map(file => file -> ("akka/" + file.getName))
       }
     )
 
+  lazy val aggregatedProjects: Seq[ProjectReference] = Seq(
+    actor,
+    actorTests,
+    agent,
+    benchJmh,
+    camel,
+    cluster,
+    clusterMetrics,
+    clusterSharding,
+    clusterTools,
+    contrib,
+    distributedData,
+    docs,
+    http,
+    httpCore,
+    httpJackson,
+    httpSprayJson,
+    httpTestkit,
+    httpTests,
+    httpXml,
+    kernel,
+    multiNodeTestkit,
+    osgi,
+    parsing,
+    persistence,
+    persistenceQuery,
+    persistenceShared,
+    persistenceTck,
+    protobuf,
+    remote,
+    remoteTests,
+    samples,
+    slf4j,
+    stream,
+    streamTestkit,
+    streamTests,
+    streamTestsTck,
+    testkit,
+    typed
+  )
+
   lazy val root = Project(
     id = "akka",
     base = file("."),
-    aggregate = Seq(actor, testkit, actorTests, remote, remoteTests, camel,
-      cluster, clusterMetrics, clusterTools, clusterSharding, distributedData,
-      slf4j, agent, persistence, persistenceQuery, persistenceTck, persistenceShared,
-      kernel, osgi, docs, contrib, samples, multiNodeTestkit, benchJmh, typed, protobuf,
-      stream, streamTestkit, streamTests, streamTestsTck, parsing,
-      httpCore, http, httpSprayJson, httpXml, httpJackson, httpTests, httpTestkit
-    )
+    aggregate = aggregatedProjects
   ).settings(rootSettings: _*)
 
   lazy val akkaScalaNightly = Project(
     id = "akka-scala-nightly",
     base = file("akka-scala-nightly"),
     // remove dependencies that we have to build ourselves (Scala STM)
     // samples don't work with dbuild right now
-    aggregate = Seq(actor, testkit, actorTests, remote, remoteTests, camel,
-      cluster, clusterMetrics, clusterTools, clusterSharding, distributedData,
-      slf4j, persistence, persistenceQuery, persistenceTck, persistenceShared,
-      kernel, osgi, contrib, multiNodeTestkit, benchJmh, typed, protobuf,
-      stream, streamTestkit, streamTests, streamTestsTck, parsing,
-      httpCore, http, httpSprayJson, httpXml, httpJackson, httpTests, httpTestkit
-    )
+    aggregate = aggregatedProjects diff List(agent, docs, samples)
```

Comment:
Should that comment go also maybe (not sure that it was even relevant before this PR).